content/en/metrics/_index.md

```diff
@@ -64,29 +65,29 @@ The graphing experience is consistent whether you are using dashboards, notebook
 
 A metric query in Datadog looks like this:
 
-{{< img src="metrics/introduction/color-query2.png" alt="Example query with color-coded sections"  style="width:70%;">}}
+{{< img src="metrics/introduction/newanatomy.jpg" alt="Example query with color-coded sections"  style="width:70%;">}}
 
 You can break this query into a few steps:
 
-#### Metric name
+#### 1. Metric name
```

Comment:
We should avoid numbered steps in headings tags if possible. We could reformat these steps to not be headings, or keep them without numbers.

```suggestion
#### Metric name
```